### PR TITLE
Added meetings to app/db

### PR DESCRIPTION
**Pull request by croz1007**
Seed meeting data to db

### CODE REVIEW

File: lib/cbus_elixir/app/app.ex [hunk 1]
```diff
@@ -70,10 +72,12 @@ defmodule CbusElixir.App do
   def update_speaker(%Speaker{} = speaker, attrs) do
     speaker
     |> Speaker.changeset(attrs)
+
```
Comment (mdarby) [inline lib/cbus_elixir/app/app.ex]: Why the blank lines? Try running `mix format`

File: lib/cbus_elixir_web/views/meeting_view.ex [hunk 2]
```diff
@@ -0,0 +1,17 @@
+defmodule CbusElixirWeb.MeetingView do
+  use CbusElixirWeb, :view
+  alias CbusElixirWeb.MeetingView
+
```
Comment (mdarby) [inline lib/cbus_elixir_web/views/meeting_view.ex]: Pretty sure we can nix this

**Review by mdarby — DISMISSED** [review]

File: lib/cbus_elixir/app/app.ex (hunk 1 above)
Comment (croz1007) [reply lib/cbus_elixir/app/app.ex]: No clue. I removed the meeting stuff and surrounding git comments about conflicts.
`mix format` did not work globally. had to provide the specific file. Updated and pushed.

File: lib/cbus_elixir_web/views/meeting_view.ex (hunk 2 above)
Comment (croz1007) [reply lib/cbus_elixir_web/views/meeting_view.ex]: Nix'd

**Review by mdarby — APPROVED** [review]

**Pull request merged**